docs: retroactive MCP release notes for Sync 2.1.7 and Portal 1.17.1 (#1951)
TykTechnologies/tyk-docs: developer-support/release-notes/portal.mdx

If you are upgrading to 1.17.1, please follow the detailed [upgrade instructions](#upgrading-tyk).

<Note>

> If you are also upgrading Tyk Dashboard to 5.13.0, upgrade to EDP v1.17.1 at the same time. Dashboard 5.13.0 introduces MCP Proxy definitions and policies that grant access to them. Without this release, those policies will not be filtered from the developer catalogue and may be inadvertently published to portal users. See [MCP Gateway upgrade considerations](/developer-support/upgrading#mcp-upgrade-considerations) for the full component deployment order.

Reviewer: ```suggestion
**If you are upgrading Tyk Dashboard to v5.13.0, we strongly recommend that you upgrade to Developer Portal v1.17.1 at the same time.**

Tyk Dashboard v5.13.0 introduces [MCP Proxy](/ai-management/mcp-gateway/managing-mcp-proxies) support. Products granting access to MCP Proxies are not currently supported from Developer Portal.

With the introduction of MCP Proxy support in the Gateway and Dashboard, however, Policies may be created on the Dashboard that grant access to MCP proxies. Prior to Developer Portal v1.17.1, such Policies would be visible in the Developer Portal and could be inadvertently published to Portal users.

See [MCP Gateway upgrade considerations](/developer-support/upgrading#mcp-upgrade-considerations) for the full component deployment order.
```

If you are upgrading to 1.17.2, please follow the detailed [upgrade instructions](#upgrading-tyk).

<Note>
**If you are upgrading Tyk Dashboard to v5.13.0, we strongly recommend that you upgrade to Developer Portal v1.17.2 at the same time.**

> Tyk Dashboard v5.13.0 introduces [MCP Proxy](/ai-management/mcp-gateway/managing-mcp-proxies) support. Products granting access to MCP Proxies are not currently supported from Developer Portal.

Reviewer 2: ```suggestion
Tyk Dashboard v5.13.0 introduces [MCP Proxy](/ai-management/mcp-gateway/managing-proxies) support. Products granting access to MCP Proxies are not currently supported from Developer Portal.
```

Updated the base Docker images for Tyk Portal to Debian 13 (Trixie) to address multiple vulnerabilities in the underlying operating system.
</Accordion>

> <Accordion title='MCP Proxy policies excluded from the developer catalogue'>

Reviewer: What do you mean here by "developer catalogue"? This is not a Portal term.

> Policies containing MCP access rights are no longer displayed in the developer catalogue. Previously, these policies appeared as API Products or Plans in the catalogue but could not be consumed correctly because the portal has no native MCP Proxy support.

Reviewer: Plans are not related to access rights, so I'm not sure why they are mentioned here.

The "Catalog" is a curated collection of API Products and Plans -> MCP policies should magically appear in these.

I'm guessing you mean that Dashboard Policies granting access to MCP Proxies would be synchronised to the Portal along with all other Policies and hence visible and could accidentally be published in Catalogs?

> MCP Proxy policies will appear in the catalogue once full portal support for MCP is available in a future release.

Reviewer: This is not the terminology we use! We do not talk about Policies with respect to Portal.

```suggestion
Support for the publication of MCP Proxies on the Developer Portal and management of access requests for Products containing these will be available in a future release.
```

If you are upgrading to 1.17.1, please follow the detailed [upgrade instructions](#upgrading-tyk).

<Note>
**If you are upgrading Tyk Dashboard to v5.13.0, we strongly recommend that you upgrade to Developer Portal v1.17.1 at the same time.**

Tyk Dashboard v5.13.0 introduces [MCP Proxy](/ai-management/mcp-gateway/managing-mcp-proxies) support. Products granting access to MCP Proxies are not currently supported from Developer Portal.

With the introduction of MCP Proxy support in the Gateway and Dashboard, however, Policies may be created on the Dashboard that grant access to MCP proxies. Prior to Developer Portal v1.17.1, such Policies would be visible in the Developer Portal and could be inadvertently published to Portal users.

See [MCP Gateway upgrade considerations](/developer-support/upgrading#mcp-upgrade-considerations) for the full component deployment order.

> </Note>

Reviewer 2: @andyo-tyk @challiscj should we move this to 1.17.2 portal RNs?

Reviewer: Yes please, no-one should be using 1.17.1.

Updated the base Docker images for Tyk Portal to Debian 13 (Trixie) to address multiple vulnerabilities in the underlying operating system.
</Accordion>

<Accordion title='MCP Proxies excluded from the Developer Portal'>
MCP Proxies are not surfaced in the Developer Portal as the Portal has no native MCP support.

Full support for MCP Proxies in the Developer Portal will be available in a future release.

> </Accordion>

Reviewer 2: We have added the actual support in [1.17.2](https://tyk.io/docs/developer-support/release-notes/portal#avoid-accidental-exposure-of-mcp-proxies), but updating 1.17.1

Reviewer: No, we haven't added the full support yet. We've just implemented a filter so MCP proxy access policies don't appear as API Products in the Portal (where they would cause confusion).

> Policies granting access to MCP Proxies are no longer synchronised to the Developer Portal as API Products. Prior to this release, such Policies would be included in synchronisation with the Dashboard and could inadvertently appear in Catalogs, where they would fail to operate correctly.

Reviewer 2: ```suggestion
MCP Proxies are not surfaced in the Developer Portal as the Portal has no native MCP support.
```

> Support for the publication of MCP Proxies on the Developer Portal and management of access requests for Products containing these will be available in a future release.

Reviewer 2: ```suggestion
Full support for MCP Proxies in the Developer Portal will be available in a future release.
```
